### PR TITLE
feat: #1160 submit delegated admin form

### PR DESCRIPTION
**Pull request by MCatherine1994**
refs: #1160

- Allow submit for adding a delegated admin
- Added popup confirmation. We can adjust the dialog style in ticket #1191
- Refactor the method to set notifications to make it reusable for both `GrantAccess` and `GrantDelegatedAdmin`
- Fix the error handling when the case backend is not available, return error message in notification

### CODE REVIEW

File: frontend/src/components/grantaccess/GrantDelegatedAdmin.vue [hunk 1]
```diff
@@ -89,15 +99,93 @@ const areVerificationsPassed = () => {
...
+    let errorList: string[] = [];
+    let errorCode = ErrorCode.Default;
+    const data = toRequestPayload(formData.value);
 
+    try {
+        const returnResponse =
+            await AdminMgmtApiService.delegatedAdminApi.createAccessControlPrivilegeMany(
+                data
+            );
+        returnResponse.data.forEach((response) => {
```
Comment (MCatherine1994) [inline frontend/src/components/grantaccess/GrantDelegatedAdmin.vue:115]: The POST delegated admin method returns a list of result for each forest client numbers as described in ticket #1089.  We need to check the response status for each forest client number.

File: frontend/src/components/managePermissions/table/DelegatedAdminTable.vue [hunk 2]
```diff
@@ -81,7 +81,7 @@ const delegatedAdminSearchChange = (newvalue: string) => {
                 :globalFilterFields="[
                     'user.user_name',
                     'user.user_type.description',
-                    'role.role_name.role_name',
+                    'role.role_name',
```
Comment (MCatherine1994) [inline frontend/src/components/managePermissions/table/DelegatedAdminTable.vue:84]: fix the filter issue in the delegated admin table for role column

File: frontend/src/components/grantaccess/GrantDelegatedAdmin.vue [hunk 3]
```diff
@@ -183,3 +271,28 @@ const handleSubmit = async () => {
         </div>
     </VeeForm>
 </template>
+<style lang="scss">
+@use '@bcgov-nr/nr-theme/design-tokens/light-buttons.scss' as lightButton;
+@use 'sass:map';
+
+.dialog-accept-button {
```
Comment (MCatherine1994) [inline frontend/src/components/grantaccess/GrantDelegatedAdmin.vue:278]: the dialog popup accept button now previously was set to be "red" color for the delete option in the common repo, set it to be our regular primary blue here

File: frontend/src/components/managePermissions/table/ApplicationAdminTable.vue [hunk 4]
```diff
@@ -68,7 +68,6 @@ const deleteAdmin = (admin: FamAppAdminGetResponse) => {
         header: 'Remove Access',
         rejectLabel: 'Cancel',
         acceptLabel: 'Remove',
-        acceptClass: 'p-button-danger',
```
Comment (MCatherine1994) [inline frontend/src/components/managePermissions/table/ApplicationAdminTable.vue:71]: this is not in use actually

**Review by NickSaglioni — APPROVED** [review]
Looks good to me. Thanks Catherine!

**Review by J0taFerreira — APPROVED** [review]
Great job! Thanks, Catherine!

**Pull request merged**